Client/Frontend/Browser/Onboarding/Welcome/WelcomeBraveBlockedAdsController.swift

```diff
@@ -11,69 +11,184 @@ import BraveShared
 import Shared
 
 class WelcomeBraveBlockedAdsController: UIViewController, PopoverContentComponent {
-  private let label = UILabel().then {
+  
+  private struct UX {
+    static let detailsViewEdgeInset = UIEdgeInsets(equalInset: 15)
+    static let contentViewEdgeInset = UIEdgeInsets(top: 25, left: 15, bottom: 15, right: 15.0)
+    static let instructionsEdgeInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 15.0)
+  }
+  
+  let contentStackView = BlockedAdsStackView(edgeInsets: UX.contentViewEdgeInset, spacing: 15.0).then {
+    $0.axis = .vertical
+    $0.alignment = .fill
```

Comment:
nit: redundant

Reply:
At this particular case it is not because BlockedAdsStackView is setting alignment as center.